frontend/src/features/review/components/icons/PenToolIcon.tsx

```
@@ -0,0 +1,18 @@
+const PenToolIcon = ({ active = true }: { active?: boolean }) => {
+  return (
+    <svg
+      width='16'
+      height='16'
+      viewBox='0 0 20 20'
+      fill='none'
+      xmlns='http://www.w3.org/2000/svg'
+    >
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)